### PR TITLE
Add semantic-release

### PR DESCRIPTION
**Pull request by esloch**

### CODE REVIEW

**Comment by xmnlab** [comment]
let's have a meeting tomorrow so we can do the review together pls
could you send me an invitation for that pls? thanks

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -1,48 +1,41 @@
...
-on:
-  workflow_dispatch:
+name: release
+
+on: [workflow_dispatch]
 
 jobs:
-  release:
-    name: Release
+  build:
```
Comment (xmnlab) [inline .github/workflows/release.yaml:6]: keep release job name

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -1,48 +1,41 @@
...
-    concurrency:
-      group: ci-${{ github.ref }}
-      cancel-in-progress: true
-
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
+    - uses: actions/checkout@v2
```
Comment (xmnlab) [inline .github/workflows/release.yaml:14]: keep the previous version for the checkout step

**Review by xmnlab — CHANGES_REQUESTED** [review]
just added 2 comments here. other than that, LGTM!

File: .github/workflows/release.yaml (hunk 1 above)
Comment (esloch) [reply .github/workflows/release.yaml]: I copied from this [job-name](https://github.com/osl-incubator/poc-semantic-release/blob/main/.github/workflows/release.yaml)

**Pull request merged**